azuread/resource_group.go

```diff
@@ -61,6 +78,19 @@ func resourceGroupCreate(d *schema.ResourceData, meta interface{}) error {
 	}
 	d.SetId(*group.ObjectID)
 
+	// Add members if specified
+	if v, ok := d.GetOk("members"); ok {
+		members := tf.ExpandStringSlicePtr(v.(*schema.Set).List())
+
+		for _, memberUuid := range *members {
```

Comment:
I think it might be nice to have another function `graph.GroupAddMembers()` that wraps this functionality around the singular function.